src/styles/styles.ts

```diff
@@ -3974,6 +3983,117 @@ const styles = (theme: ThemeColors) =>
         singleOptionSelectorCircle: {
             borderColor: theme.icon,
         },
+
+        videoPlayerControlsContainer: {
+            position: 'absolute',
+            bottom: 8,
+            left: 8,
+            right: 8,
+            backgroundColor: '#061B09CC',
```

Comment:
Can we reuse the `darkAppBackground` color?
```suggestion
            backgroundColor: `#${colors.darkAppBackground}CC`,
```

I'm not sure about the opacity (`CC`) part. But at least we should change the color, in case someone changes it.

We should maybe even move it to the `src/styles/themes/default.ts` file.

Reply:
Good idea, I will add it ti `default.ts`